README.md

```diff
@@ -554,7 +564,8 @@ The following setting is optional and allows you to configure image registry cre
 * `settings.container-registry.credentials`: An array of container images registry credential settings. Each element specifies the registry and the credential information for said registry.
 The credential fields map to [containerd's registry credential fields](https://github.com/containerd/containerd/blob/v1.6.0/docs/cri/registry.md#configure-registry-credentials), which in turn map to the fields in `.docker/config.json`.
 It is recommended to programmatically set these settings via `apiclient` through the Bottlerocket control container and/or custom host-containers.
-  * An example `apiclient` call to set registry credentials for `gcr.io` and `docker.io` looks like this:
+
+  An example `apiclient` call to set registry credentials for `gcr.io` and `docker.io` looks like this:
```

Comment:
Same here

Reply:
This is definitely intentional. We want the example to be text under the bullet item and not outside of it so that it stays connected to the setting that it is giving an example for.

Reply:
Ah I see that now - disregard!